core/remote/cozy.js

```diff
@@ -111,6 +108,8 @@ class RemoteCozy {
     autoBind(this)
   }
 
+  // FIXME: setup can be done multiple times if getClient() is called multiple times concurrently.
+  // Use lock?! Separate setup from getter ?!
   async getClient() /*: Promise<CozyClient> */ {
```

Comment:
This won't help with the issue I stumbled upon. The setup needs to be called before and, most of all, outside `getClient()`.
But I didn't find the right place to do so for now (I did not spend a lot of time on it either).

It might be easier to leave it as such until we replace `cozy-client-js` with `cozy-client` completely (which should not be too far in the future).